front/src/NotificationWindow.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color : ${props => props.backgroundColor}
+  border-style: solid;
+  border-size: 4px;
+  width: 90%;
+  position: relative;
+  margin-left:auto;
+  margin-right:auto;
```

Comment:
`margin: 0 auto;` ?

Reply:
Didn't know that was a thing, will change it (if it works).